src/test/java/com/softserveinc/ita/dkrutenko/RozetkaTests.java

```diff
@@ -2,17 +2,19 @@
 
 import com.softserveinc.ita.dkrutenko.utils.runners.TestRunner;
 import org.testng.Assert;
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Test;
+import org.testng.annotations.*;
 
-public class rozetkaTests extends TestRunner {
+public class RozetkaTests extends TestRunner {
+
+    @BeforeClass
+    private void setUrl() {
```

Comment:
I think, that driver.get() open url, so maybe better rename method to something like openUrl, and maybe open rozetka for every test, try to use @BeforeMethod annotation

Reply:
@BeforeMethod
private void openUrl() {
driver.get(rozetkaUrl);
}

Reply:
it works for every rozetka test in RozetkaTests class. I dont need to open url in testrunner because i have softserve tests